tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -206,3 +206,73 @@ def test_positive_resource_vm_power_management(
             session.computeresource.vm_poweron(name, rhev_data['vm_name'])
         assert session.computeresource.vm_status(
             name, rhev_data['vm_name']) is not status
+
+
+@skip_if_bug_open('bugzilla', 1636067)
+@tier2
+@parametrize('version', [True, False])
```

Comment:
could you use `ids` parameter to get sane test names?
e.g. `test_positive_VM_import[v3]`, `test_positive_VM_import[v4]`